dbt_semantic_interfaces/implementations/metric.py

```diff
@@ -228,3 +238,31 @@ def input_metrics(self) -> Sequence[PydanticMetricInput]:
             return (self.type_params.numerator, self.type_params.denominator)
         else:
             assert_values_exhausted(self.type)
+
+    @staticmethod
+    def all_input_measures_for_metric(
```

Comment:
should this method be in the protocol?

Reply:
I think not. I don't think we want to require implementing this logic any place that implements a metric.